cogs5e/initiative/combatant_builders.py

```diff
@@ -0,0 +1,115 @@
+from d20 import roll
+from disnake.ext import commands
+from cogs5e.models.sheet.base import BaseStats, Levels, Saves, Skill, Skills
+from cogs5e.models.sheet.resistance import Resistances
+from utils.constants import STAT_ABBR_MAP
+from utils.argparser import argparse
+from cogs5e.initiative.effects.passive import resolve_check_advs, resolve_save_advs
+from .utils import create_combatant_id
+from . import (
+    Combatant,
+    utils,
+)
+
+async def add_builder(ctx, combat, name, args):
```

Comment:
IMO the builder function should stay synchronous, as the async signature implies that the function has some side effects. See comments below for some tips to make this sync